### ISSUE TITLE
API documentation link is broken

### ISSUE THREAD

**Issue by arichiardi** [michaelklishin/langohr#108]
Hey Michael, thank you for the project!

I opened a small issue just to report that I tried to click on the API documentation link but it seems to be broken.

Sorry and thank you!

**Comment by michaelklishin** [comment]
Yes, all of ClojureWerkz API doc sites were hosted on a Digital Ocean droplet that is so old it could not upgrade (from 2012 or so?), so before I migrate them away, this will remain the case. There is no ETA.
